### ISSUE TITLE
Can't upgrade from 0.1.10 (mistakenly reported as 0.1.1?) to 0.2.0

### ISSUE THREAD

**Issue by ishioni** [tensorchord/pgvecto.rs#397]
I've been fighting this for over a week with varying results, sanitizing data and whatnot. I'm currently at trying to do this

```
BEGIN;

CREATE SCHEMA IF NOT EXISTS vectors;
ALTER DATABASE immich SET search_path TO "$user", public, vectors;
SET search_path TO "$user", public, vectors;

UPDATE pg_catalog.pg_extension SET extversion = '0.1.1' WHERE extname = 'vectors';
UPDATE pg_catalog.pg_extension SET extrelocatable = true WHERE extname = 'vectors';
ALTER EXTENSION vectors SET SCHEMA vectors;
UPDATE pg_catalog.pg_extension SET extrelocatable = false WHERE extname = 'vectors';
ALTER EXTENSION vectors UPDATE TO '0.2.0';

COMMIT;
```

And ending up with this 
```
COMMIT;
BEGIN
CREATE SCHEMA
ALTER DATABASE
SET
UPDATE 1
UPDATE 1
ALTER EXTENSION
UPDATE 1
ERROR:  could not find function "ai_embedding_vector_wrapper" in file "/usr/lib/postgresql/16/lib/vectors.so"
ROLLBACK
```

```
immich=# SELECT * FROM pg_catalog.pg_extension ;
  oid   |    extname    | extowner | extnamespace | extrelocatable | extversion | extconfig | extcondition 
--------+---------------+----------+--------------+----------------+------------+-----------+--------------
  13541 | plpgsql       |       10 |           11 | f              | 1.0        |           | 
  19714 | uuid-ossp     |       10 |         2200 | t              | 1.1        |           | 
  91167 | cube          |       10 |         2200 | t              | 1.5        |           | 
  91256 | earthdistance |       10 |         2200 | t              | 1.1        |           | 
 112973 | vectors       |       10 |         2200 | f              | 0.1.1      |           | 
 immich=# select * from pg_available_extensions where name = 'vectors';
  name   | default_version | installed_version |                                           comment                                            
---------+-----------------+-------------------+----------------------------------------------------------------------------------------------
 vectors | 0.2.0           | 0.1.1             | vectors: Vector database plugin for Postgres, written in Rust, specifically designed for LLM
 ```

This is on cnpg going from ghcr.io/bo0tzz/cnpgvecto.rs:16.1-v0.1.13 (initially created with 0.1.10) to ghcr.io/tensorchord/cloudnative-pgvecto.rs:16.2-v0.2.0

**Comment by ishioni** [comment]
Trying with 0.1.11 instead of 0.1.1 gives
ERROR:  type "vectorindexinfo" does not exist
0.1.12 or 0.1.13
ERROR:  type "vectorindexstat" does not exist

**Comment by VoVAllen** [comment]
Hi, seems you've met the same problem as https://github.com/tensorchord/pgvecto.rs/issues/380. We're preparing a fix at https://github.com/tensorchord/pgvecto.rs/pull/395 and targeting to make a new release today to fix this problem

**Comment by VoVAllen** [comment]
Fixed in 0.2.1.

Ref: https://github.com/tensorchord/pgvecto.rs/discussions/405
